core/common/src/main/java/alluxio/grpc/GrpcChannelBuilder.java

```diff
@@ -188,11 +188,6 @@ public GrpcChannelBuilder setPoolingStrategy(GrpcManagedChannelPool.PoolingStrat
     return this;
   }
 
-  public GrpcChannelBuilder setExecutor(Executor executor) {
```

Comment:
We don't want this API anymore?

Reply:
I just didn't want to expose it as it's not used. We can expose it when it become useful.